packages/subscription-manager/src/components/SubscriptionManagerContainer/SubscriptionManagerContainer.tsx

```diff
@@ -12,6 +13,43 @@ export type SubscriptionManagerContainerProps = {
 	children?: React.ReactNode;
 };
 
+const getEmailAddress = () => {
+	const subkey = getSubkey();
+
+	if ( ! subkey ) {
+		return null;
+	}
+
+	const decodedSubkeyValue = decodeURIComponent( subkey );
+
+	const firstPeriodIndex = decodedSubkeyValue.indexOf( '.' );
+	if ( firstPeriodIndex === -1 ) {
+		return null;
+	}
+
+	const emailAddress = decodedSubkeyValue.slice( firstPeriodIndex + 1 );
+	return emailAddress;
+};
```

Comment:
I see this function as a hook like `useSubscriberEmailAddress`, using useMemo to avoid processing the string on every render. It could be either placed on this file or on  a hooks.ts file (in this case it could be reused by other components if needed). What do you think?

Reply:
Thanks, Paulo. That makes sense. I have applied the changes.